src/components/network-modification-tree.js

```diff
@@ -222,7 +249,18 @@ const NetworkModificationTree = ({
                     showInteractive={false}
                 >
                     <CenterGraphButton selectedNode={selectedNode} />
+                    <ControlButton
+                        onClick={() =>
+                            setIsMinimapOpen(isMinimapOpen ? false : true)
```

Comment:
Use 
`setIsMinimapOpen(!isMinimapOpen)`
instead ?

Reply:
or setIsMinimapOpen( oldval => !oldVal);  and make it a callback with no dependency ? 
const toggleMinimapOpen = useCallback( () => setIsMinimapOpen( oldval => !oldVal));

Reply:
I went for the toggle callback method